marurevi/Blog-app
# Blog App: validations and model specs

## Validations and Model specs
In this project I implemented the following features
- Install and setup spec enviroment
- implement workflow to check tests with rspec
- Add Validations:
  - **For the User model:**
    * Name must not be blank.
    * PostsCounter must be an integer greater than or equal to zero.
  - **For the Post model:**
    * Title must not be blank.
    * Title must not exceed 250 characters.
    * CommentsCounter must be an integer greater than or equal to zero.
    * LikesCounter must be an integer greater than or equal to zero.
    * Add unit specs for all of your models' methods and validations.
  
![Tests](https://user-images.githubusercontent.com/92180054/184289929-181a1906-5047-4954-b113-51035361a0fb.png)
